widget-missing-in-kendoui-professional-package.md

```diff
@@ -0,0 +1,97 @@
+
+---
+title: Widgets from the Kendo UI Core/Professional Package Don't Load
+description: After installing Kendo UI via the Package Manager, widget XXXX doesn't work
```

Comment:
The widgets never worked, so I can't use "no longer". It is a certain widget from a quite long list of wigets, depending what the customer wants to show.

Tickets usually go like "Grid/Scheduler/MultiSelect/etc. doesn't work in my app (even though I enabled Kendo UI Professional)". At the same time, other widgets work.

That is why I thought I should use something that could be replaced by any widget name from the list of widgets that aren't included in our packages.